packages/lesswrong/lib/collections/posts/fragments.ts

```diff
@@ -163,6 +163,7 @@ registerFragment(`
       htmlHighlight
       wordCount
       version
+      plaintextDescription
```

Comment:
Hmm, so this makes postlist actually pull more data over the wire. It's now pulling an extra 2000 characters for every post. I'm not sure, but it might also be causing the resolver to calculate that plaintext description dynamically. If so, again I'm not sure, but that might make the resolving of data for long lists (like in allPosts) take noticeably longer.